recipes/emsdk/all/conanfile.py

```diff
@@ -0,0 +1,124 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class EmSDKConan(ConanFile):
+    name = "emsdk"
+    description = "Emscripten is an Open Source LLVM to JavaScript compiler"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/kripken/emscripten"
+    topics = ("conan", "emsdk", "emscripten", "sdk")
+    license = "MIT"
+    settings = "os"
+
+    short_paths = True
+
+    @ property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def package_id(self):
+        del self.info.settings.os
```

Comment:
No need to modify anything here. We likely need different binaries (different package ID) for Windows, Linux, Macos,  M1,...